storage/watchable_store_test.go

```diff
@@ -94,4 +94,24 @@ func TestUnsafeAddWatching(t *testing.T) {
 		// when putting the same watch twice"
 		t.Error(`error = nil, want "put the same watch twice"`)
 	}
+
+	// test the map operation
+	ws := &watching{
```

Comment:
can we change this to a simple for loop with the previous one? like add three watching on the same key?